elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java

```diff
@@ -727,6 +728,46 @@ public void testPathToJAASConfigFileDoesNotExist() throws Exception {
         Assert.assertTrue(response.asString().contains("JAAS configuration file does not exist."));
     }
 
+    @Test
+    public void testDistributedRealm() throws Exception {
+        KernelServices services = super.createKernelServicesBuilder(new TestEnvironment()).setSubsystemXmlResource("realms-test.xml").build();
+        if (!services.isSuccessfulBoot()) {
+            Assert.fail(services.getBootError().toString());
+        }
+
+        ServiceName serviceName = Capabilities.SECURITY_REALM_RUNTIME_CAPABILITY.getCapabilityServiceName("DistributedRealmOne");
```

Comment:
> DistributedRealmOne

Can you use more descriptive names please? For example DistributedRealmNoUnavailable, DistributedRealmFirstUnavailableFailing, DistributedRealmFirstUnavailableIgnored.